.github/workflows/ci-suite.yml

```diff
@@ -160,6 +160,8 @@ jobs:
   dox-build:
     name: Build Doxygen Site
     runs-on: ubuntu-latest
+    env:
+      TGS_RELEASE_NOTES_TOKEN: ${{ secrets.TGS_RELEASE_NOTES_TOKEN }}
```

Comment:
Sorry, I obviously wasn't paying attention when I first looked.

We already have `TGS_RELEASE_NOTES_TOKEN` as an alias for `secrets.DEV_PUSH_TOKEN` defined in the global environment. You were right the first time.

Could you remove the environment definition please?

Reply:
I discarded the latest commit